docs/self-managed/backup-restore/backup-and-restore.md

```diff
@@ -6,7 +6,7 @@ keywords: ["backup", "backups"]
 ---
 
 :::note
-This API is subject to change.
+This release introduces breaking changes for Operate and Tasklist. See [backup and restore Operate and Tasklist data](./operate-tasklist-backup.md) for details.
```

Comment:
@RomanJRW and @Zelldon I'm noting Operate and Tasklist breaking changes here, but are there any for Optimize or Zeebe in the `next` version of docs? Are these still subject to change?

Reply:
@PHWaechtler - can you answer here please for the Optimize side

Reply:
I guess @deepthidevaki knows best regarding backup

Reply:
Yes I would say the same applies for Optimize as we've made some of the same changes as operate/tasklist, eg renaming `/backup` to `/backups`

Reply:
@christinaausley There are similar API changes in Zeebe as well. The new API is already documented. But we didn't document it as a breaking change as it was an experimental feature in 8.1.

Reply:
Thanks all!